error_code_report-0.1.0.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.exasol.com/error_code_report-0.1.0.json",
+  "type": "object",
+  "title": "Exasol error code report schema.",
+  "description": "This schema defines the structure of a report about error codes in an Exasol project.",
+  "additionalProperties": false,
+  "required": [
+    "projectName",
+    "projectVersion",
+    "errorCodes"
+  ],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "description": "Reference to the schema. Setting this helps your IDE to assist you while writing the definition.",
+      "enum": [
+        "https://schemas.exasol.com/error_code_report-0.1.0.json",
+        "./error_code_report-0.1.0.json"
+      ]
+    },
+    "projectName": {
+      "type": "string",
+      "title": "Name of the project.",
+      "description": "Typically the GitHub repository name."
+    },
+    "projectVersion": {
+      "type": "string",
+      "title": "Version tag of the reported release."
+    },
+    "errorCodes": {
+      "type": "object",
+      "patternProperties": {
+        "^[WFE]-[A-Z][A-Z0-9]*(-[A-Z][A-Z0-9]*)?-\\d$": {
```

Comment:
If I interpret the Regex correct, then it only allows `F-ABC-ABC-100` but not `F-ABC-ABC-ABC-100`, from the the UDF, I know that I need the later one because it already has quite many components and these sometimes also have subcomponents. 

`^[WFE]-[A-Z][A-Z0-9]*(-[A-Z][A-Z0-9]*)*-\\d$`

I used in my scripts the following one:

`[FEW]-[A-Z0-9]+(-[A-Z0-9]+)*-[0-9]+`

Reply:
This is however not compliant with https://intranet.srv.exasol.com/display/RD/Unified+error+handling.
@redcatbear should we change it?